src/Ext/Building/Hooks.cpp

```diff
@@ -115,5 +117,168 @@ DEFINE_HOOK(0x43FE73, BuildingClass_AI_FlyingStrings, 0x6)
 		}
 	}
 
+	return 0;
+}
+
+DEFINE_HOOK(0x4502F4, BuildingClass_Update_Factory, 0x6)
+{
+	GET(BuildingClass*, pBuilding, ESI);
+	HouseClass* pOwner = pBuilding->Owner;
```

Comment:
I would leave GET()'s alone.

```suggestion
	GET(BuildingClass*, pBuilding, ESI);
	
	HouseClass* pOwner = pBuilding->Owner;
```